@kiva/kv-components/tests/fixtures/mockFeedActivityData.js

```diff
@@ -1,212 +1,155 @@
+								user: {
+									publicLenderId: 'Sarah1234',
+									name: 'Sarah',
+									image: 'https://www-0.development.kiva.org/img/s100/6b1a24092be3aaa22216874e644a4acf.jpg', // eslint-disable-line max-len
+								},
+								kind: 'comment',
+								activity_id: 'efb2dbfe-c12a-11ee-9fbc-065afce7d41d',
+								text: 'child comment reply',
+								parent: '6384df77-d7c4-4ea9-b1b9-ef1c76a054a2',
+								latest_children: {},
+								children_counts: {},
 							},
-						},
-						parent: '',
-						latest_children: {},
-						children_counts: {},
-						is_liked: true,
+						],
 					},
-					{
-						created_at: '2024-02-01T20:06:46.651764Z',
-						updated_at: '2024-02-01T20:06:46.651764Z',
-						id: 'e1db4420-159e-4ba2-aab9-704d1cc56dae',
-						user: {
-							created_at: '2024-02-28T19:09:54.093757Z',
-							updated_at: '2024-02-28T19:09:54.093757Z',
-							data: {
-								publicLenderId: 'Todd1234',
-								name: 'Todd',
-								image: 'https://www-0.development.kiva.org/img/s100/6b1a24092be3aaa22216874e644a4acf.jpg', // eslint-disable-line max-len
+					children_counts: {
+						comment: 1,
+					},
+				},
+				{
+					created_at: '2024-02-28T20:04:21.445272Z',
+					updated_at: '2024-02-28T20:04:21.445272Z',
+					id: '290a6c07-143a-416e-b016-ac0d9ac8ad90',
+					user: {
+						publicLenderId: 'Sophie1234',
+						name: 'Sophie',
+						image: 'https://www-0.development.kiva.org/img/s100/6b1a24092be3aaa22216874e644a4acf.jpg', // eslint-disable-line max-len
+					},
+					kind: 'comment',
+					activity_id: 'efb2dbfe-c12a-11ee-9fbc-065afce7d41d',
+					text: 'second child comment',
+					parent: '6384df77-d7c4-4ea9-b1b9-ef1c76a054a2',
+					latest_children: {
+						like: [
+							{
+								created_at: '2024-02-28T20:04:21.445272Z',
+								updated_at: '2024-02-28T20:04:21.445272Z',
+								id: '290a6c07-143a-416e-b016-ac0d9ac8ad99',
+								user: {
+									created_at: '2024-02-28T19:09:54.093757Z',
+									updated_at: '2024-02-28T19:09:54.093757Z',
+									publicLenderId: 'Jess1234',
+									name: 'Jess',
+									image: 'https://www-0.development.kiva.org/img/s100/6b1a24092be3aaa22216874e644a4acf.jpg', // eslint-disable-line max-len
+								},
+								kind: 'like',
+								activity_id: 'efb2dbfe-c12a-11ee-9fbc-065afce7d41d',
+								parent: '6384df77-d7c4-4ea9-b1b9-ef1c76a054a2',
+								latest_children: {},
+								children_counts: {},
 							},
-						},
-						kind: 'comment',
-						activity_id: 'efb2dbfe-c12a-11ee-9fbc-065afce7d41d',
-						data: {
-							text: 'comment test forever!',
-						},
-						parent: '',
-						latest_children: {},
-						children_counts: {},
+						],
 					},
-				],
-			},
-			latest_reactions_extra: {
-				comment: {
-					next: '',
+					children_counts: {},
 				},
-			},
-			object: 'test-challenge-id',
-			origin: null,
-			target: '',
-			time: '2024-02-01T17:54:24.932762',
-			verb: 'CHALLENGE',
+			],
+		},
+		children_counts: {
+			comment: 1,
```

Comment:
Do we have more than 1 comment here?

Reply:
We can largely ignore this data for now, I just needed to get the tests to pass. There just isn't a great way to test these components locally with the live service. More changes coming when I know the data better!